pulsar-core/src/event.rs

```diff
@@ -164,25 +165,33 @@ pub enum Payload {
         len: usize,
         is_tcp: bool,
     },
-    MalwareDetection {
-        score: f32,
-        #[validatron(skip)]
-        tags: Vec<String>,
-    },
-    RuleEngineDetection {
-        #[validatron(skip)]
-        rule_name: String,
-        #[validatron(skip)]
-        payload: Box<Payload>,
-    },
-    AnomalyDetection {
-        score: f32,
-    },
+    Empty,
```

Comment:
What if a module wants to send a custom event which is not a threat?

Reply:
the same as before, we have to choose how to implement the custom type we neved did